xbmc/playlists/PlayListM3U.cpp

```diff
@@ -69,6 +69,10 @@ bool CPlayListM3U::Load(const std::string& strFileName)
   int iStartOffset = 0;
   int iEndOffset = 0;
 
+  bool utf8 = false;
+  if (URIUtils::GetExtension(strFileName) == ".m3u8")
+    utf8 = true;
```

Comment:
This is appropriate to be inline
```suggestion
  bool isUtf8 = URIUtils::GetExtension(strFileName) == ".m3u8";
```